and-bible/app/src/main/java/net/bible/service/format/osistohtml/taghandler/BookmarkMarker.java

```diff
@@ -40,18 +36,14 @@ public BookmarkMarker(OsisToHtmlParameters parameters, VerseInfo verseInfo) {
 	public List<String> getBookmarkClasses() {
 		if (bookmarkStylesByBookmarkedVerse !=null && parameters.isShowBookmarks()) {
 			if (bookmarkStylesByBookmarkedVerse.containsKey(verseInfo.currentVerseNo)) {
-				final List<BookmarkStyle> bookmarkStyles = bookmarkStylesByBookmarkedVerse.get(verseInfo.currentVerseNo);
-				if (bookmarkStyles==null || bookmarkStyles.isEmpty()) {
-					return Collections.singletonList(parameters.getDefaultBookmarkStyle().name());
-				} else {
-					return getStyleNames(bookmarkStyles);
-				}
+				final Set<BookmarkStyle> bookmarkStyles = bookmarkStylesByBookmarkedVerse.get(verseInfo.currentVerseNo);

```

Comment:
The default bookmark style code here has been deleted.  Default bookmark still works but I don't know how and osisParams.getDefaultBookmarkStyle() is not used any more.  If this is fine then maybe just add a comment and remove the redundant code.

Reply:
removed obsolete getDefaultBookmarkStyle